explorer/app/config/type.ts

```diff
@@ -0,0 +1,11 @@
+import { HeaderProps } from "./../components/Header/Header";
```

Comment:
Just curious, is `type.ts` a convention for modules that contain only a single interface or type? I have seen `constants.ts` but not `type.ts` before.

Reply:
isn't a convention, I came up with the name cuz I didn't know how to call this file and I would like to keep it separated, since we might be changing it now and then. But thinking right now, maybe model is a better name for it.

Reply:
No worries, we can adjust as we proceed if needed.